misc/docs/source/packages/obspy.io.nied.fnetmt.rst

```diff
@@ -0,0 +1,15 @@
+.. currentmodule:: obspy.io.nied.fnetmt
+.. automodule:: obspy.io.nied.fnetmt
+
+    .. comment to end block
+
+    Modules
+    -------
+    .. autosummary::
+       :toctree: autogen
+       :nosignatures:
+
+       core
```

Comment:
There is no `core` within `obspy.io.nied.fnetmt`. In fact, I'm not sure you even need this file any more.